cmd/clusterctl/examples/openstack/centos/machines.yaml.template

```diff
@@ -19,6 +19,8 @@ items:
         floatingIP: 129.114.111.153
         securityGroups:
         - default
+        userDataSecret:
+          name: master-user-data
```

Comment:
Just one more little complaint... Could you add the namespace in all the yaml files for now. Because you generate the secrets in `openstack-provider-system` rather than `machine.namespace`.

Reply:
done, good catch!